lib/services/insightsManagement/README.md

```diff
@@ -0,0 +1,38 @@
+# Microsoft Azure SDK for Node.js - Web Site Management
+
+This project provides a Node.js package that makes it easy to use the Azure Insights API. Right now it supports:
+- **Node.js version: 0.6.15 or higher**
+- **API version: 2014-04-01**
+
+## Features
+
+ - TODO
+
+## How to Install
+
+```bash
+npm install azure-arm-insights
+```
+
+## How to Use
+
+### Authentication
+
+ - TODO: Describe tokens
+
```

Comment:
Pleae take a look at the readme of storage management (https://github.com/Azure/azure-sdk-for-node/tree/master/lib/services/storageManagement2#how-to-use) and make changes to isnisghts management accordingly.